comp/core/workloadmeta/collectors/internal/ecsfargate/ecsfargate.go

```diff
@@ -232,6 +259,45 @@ func (c *collector) parseTaskContainers(
 	return taskContainers, events
 }
 
+func (c *collector) parseV4Task(task *v3or4.Task) []workloadmeta.CollectorEvent {
+	events := []workloadmeta.CollectorEvent{}
+	seen := make(map[workloadmeta.EntityID]struct{})
+
+	// We only want to collect tasks without a STOPPED status.
+	if task.KnownStatus == "STOPPED" {
```

Comment:
Should we store the string `"STOPPED" and `"RUNNING"` in constants?

Reply:
we only need `stopped`, I added it in types
https://github.com/DataDog/datadog-agent/blob/882d6e4354906f23c0124aa0d908eed0e5bcbd4a/comp/core/workloadmeta/types.go#L857